bedrock/careers/views.py

```diff
@@ -69,3 +70,14 @@ def get_context_data(self, **kwargs):
         context["related_positions"] = related_positions
 
         return context
+
+    def dispatch(self, request, *args, **kwargs):
+        try:
+            return super().dispatch(request, *args, **kwargs)
+        except Http404:
+            return render(
+                self.request,
+                "careers/404.html",
+                ftl_files=["404", "careers/404"],
```

Comment:
It's a little unusual to be mixing two separate fluent files like this, as `404.ftl` is not a global Fluent file. Because we're loading `404.ftl` first in the array, it's using that file to activate locales that have translations, resulting in a mixture of languages. E.g. view: http://localhost:8080/de/careers/404/

Is it possible to set `active_locales=['en-US']` here? If not, is it worth just hard-coding the strings in this page, seeing as careers is english only?

If we do want to localize this page, I'd suggest using a single ftl file as opposed to mixing the two.